src/scripts/constants.ts

```diff
@@ -0,0 +1,522 @@
+    ],
+    name: "setEthUsdChainLinkPriceFeed",
+    outputs: [],
+    stateMutability: "nonpayable",
+    type: "function",
+  },
+  {
+    inputs: [
+      { internalType: "uint256", name: "collateralIndex", type: "uint256" },
+      { internalType: "uint256", name: "newMintFee", type: "uint256" },
+      { internalType: "uint256", name: "newRedeemFee", type: "uint256" },
+    ],
+    name: "setFees",
+    outputs: [],
+    stateMutability: "nonpayable",
+    type: "function",
+  },
+  {
+    inputs: [{ internalType: "address", name: "newGovernanceEthPoolAddress", type: "address" }],
+    name: "setGovernanceEthPoolAddress",
+    outputs: [],
+    stateMutability: "nonpayable",
+    type: "function",
+  },
+  {
+    inputs: [
+      { internalType: "uint256", name: "collateralIndex", type: "uint256" },
+      { internalType: "uint256", name: "newCeiling", type: "uint256" },
+    ],
+    name: "setPoolCeiling",
+    outputs: [],
+    stateMutability: "nonpayable",
+    type: "function",
+  },
+  {
+    inputs: [
+      { internalType: "uint256", name: "newMintPriceThreshold", type: "uint256" },
+      { internalType: "uint256", name: "newRedeemPriceThreshold", type: "uint256" },
+    ],
+    name: "setPriceThresholds",
+    outputs: [],
+    stateMutability: "nonpayable",
+    type: "function",
+  },
+  {
+    inputs: [{ internalType: "uint256", name: "newRedemptionDelayBlocks", type: "uint256" }],
+    name: "setRedemptionDelayBlocks",
+    outputs: [],
+    stateMutability: "nonpayable",
+    type: "function",
+  },
+  {
+    inputs: [
+      { internalType: "address", name: "newPriceFeedAddress", type: "address" },
+      { internalType: "uint256", name: "newStalenessThreshold", type: "uint256" },
+    ],
+    name: "setStableUsdChainLinkPriceFeed",
+    outputs: [],
+    stateMutability: "nonpayable",
+    type: "function",
+  },
+  {
+    inputs: [],
+    name: "stableUsdPriceFeedInformation",
+    outputs: [
+      { internalType: "address", name: "", type: "address" },
+      { internalType: "uint256", name: "", type: "uint256" },
+    ],
+    stateMutability: "view",
+    type: "function",
+  },
+  {
+    inputs: [{ internalType: "uint256", name: "collateralIndex", type: "uint256" }],
+    name: "toggleCollateral",
+    outputs: [],
+    stateMutability: "nonpayable",
+    type: "function",
+  },
+  {
+    inputs: [
+      { internalType: "uint256", name: "collateralIndex", type: "uint256" },
+      { internalType: "uint8", name: "toggleIndex", type: "uint8" },
+    ],
+    name: "toggleMintRedeemBorrow",
+    outputs: [],
+    stateMutability: "nonpayable",
+    type: "function",
+  },
+  {
+    inputs: [{ internalType: "uint256", name: "collateralIndex", type: "uint256" }],
+    name: "updateChainLinkCollateralPrice",
+    outputs: [],
+    stateMutability: "nonpayable",
+    type: "function",
+  },
+] as const;
+
+export const uFaucetAddress = "0xb64f2347752192F51930Ad6Ad3bEa0B3A2074fAC";
```

Comment:
Here `UbiquityPoolFacet` address is used while we should use the diamond's [address](https://github.com/ubiquity/ubiquity-dollar/wiki/24.-Smart-Contracts#diamond).

The naming could be like `diamondAddress`.